torch/utils/_sympy/value_ranges.py

```diff
@@ -119,6 +127,10 @@ def is_singleton(self) -> bool:
     def unknown(cls):
         return cls(-sympy.oo, sympy.oo)
 
+    @classmethod
+    def unknown_bool(cls):
+        return cls(sympy.false, sympy.true)
```

Comment:
It's hard for me to picture what sort of user code + compiler analysis from our side would result in a boolean value_range 🤔 any example / test to go off?

Reply:
It is very strange! Maybe I need to study the original problem in more detail